db/seeds.rb

```diff
@@ -12,3 +12,10 @@
 admin = Account.create(email: admin_email, password: SecureRandom.hex(20), level: 1, role: 'admin', confirmed_at: Time.now)
 
 puts 'Admin credentials: %s' % [admin.password]
+
+env_file = File.join(Rails.root, 'config', 'oauth_application.yml')
```

Comment:
we need to iterates over the app
I would even call it config/seed.yml

and allow other type of values likes websites.